docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+# Autopilot Pattern NFS server implementation
+
+nfsserver:
+    image: autopilotpattern/nfsserver
+    privileged: true
```

Comment:
See https://github.com/autopilotpattern/nfsserver/pull/1#discussion_r60065684

> ## This is not recommended for production use
>
> Server and client containers need `privileged` on Linux hosts (though not on Triton, which supports this securely). This may not be a solvable problem. Docker volume drivers are probably the best recommended work around. On Triton, [RFD26 will provide network shared filesystems](https://github.com/joyent/rfd/blob/master/rfd/0026/README.md) to Docker containers using Docker volume syntax.
>
> For now, consider this an experiment.